Language-Version-History.md

```diff
@@ -104,3 +104,19 @@ Features Added in C# Language Versions
 - [Expression variables in initializers and queries](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.3/expression-variables-in-initializers.md): Expression variables like `out var` and pattern variables are allowed in field initializers, constructor initializers and LINQ queries.
 -	[Tuple comparison](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.3/tuple-equality.md): Tuples can now be compared with `==` and `!=`.
 -	[Attributes on backing fields](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.3/auto-prop-field-attrs.md): Allows `[field: …]` attributes on an auto-implemented property to target its backing field.
+
+# C# 8.0 (Visual Studio 2019 version 16.3, .NET Core 3.0)
+- [Nullable reference types](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/nullable-reference-types-specification.md): express nullability intent on reference types with `?`, `notnull` constraint and annotations attributes in APIs, the compiler will use those to try and detect possibly dangerous `null` values being dereferenced or passed to unsuitable APIs.
```

Comment:
>- [](start = 0, length = 1)

Consider adding line breaks within list items.

Reply:
Adding line breaks messes up the final rendering

---
In reply to: [305054988](https://github.com/dotnet/csharplang/pull/2666#discussion_r305054988) [](ancestors = 305054988)